tests/unit_tests/data_model/test_metadata_model.py

```diff
@@ -48,7 +48,7 @@ def test_validate_schema(tmp_test_directory):
         metadata_model.validate_schema(invalid_data, schema_file)
 
 
-def test_resolve_references():
+def test__resolve_references():
```

Comment:
we usually don't use two '_' here. I think this was changed in a "search and replace"?